scripts/tests/calibnet_stateless_mode_check.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script tests the stateless mode of a forest node
+
+source "$(dirname "$0")/harness.sh"
+
+forest_init_stateless
+
+echo "Verifying the heaviest tipset to be the genesis"
+MSG=$($FOREST_CLI_PATH chain head)
+assert_eq "$MSG" $'[\n  "bafy2bzacecyaggy24wol5ruvs6qm73gjibs2l2iyhcqmvi7r7a4ph7zx3yqd4"\n]'
+
+wait_util_p2p_is_ready
+ADDRESS=$($FOREST_CLI_PATH net listen | tail -n 1)
+echo "Stateless node address: $ADDRESS"
+PEER_ID=$(echo "$ADDRESS" | cut -d "/" -f7 -z)
```

Comment:
If possible, let's use long flag names. I had to look-up what is `-z`.

Reply:
Fixed